src/bp-core/bp-core-functions.php

```diff
@@ -5094,3 +5094,17 @@ function bp_core_get_profile_completion_key() {
 
 	return 'bbprofilecompletion';
 }
+
+/**
+ * Remove profile completion widget transient.
+ *
+ * @since BuddyBoss 1.5.3
+ */
+function bp_core_profile_completion_widget_delete_transient() {
+	$transient_keys = bp_get_option( 'bp_profile_completion_widgets', array() );
```

Comment:
@chetansatasiya Transient key should not be added to the options table. We should use that as a constant or static variable 
Because every time we need to do one option query to get the name of the transient and I don't think this need to store in DB

Reply:
Agreed!